### ISSUE TITLE
Docs: nodes outside lan

### ISSUE THREAD

**Issue by jtagcat** [overnode-org/overnode#22]

**Comment by avkonst** [comment]
Could you please provide the description about the problem and what you would like to have documented?

**Comment by jtagcat** [comment]
Say I want to have one node as a VPS at Hetzner, one at K-Space, one at a VPS in US. Will it magically just work?

**Comment by avkonst** [comment]
Yes, it should work. I had nodes spread across public cloud and private
datacenter
